internal/runtime/runtime.go

```diff
@@ -38,6 +40,8 @@ type GolangRuntime struct {
 	TargetType    interface{}
 	transforms    []appcontext.AppFunction
 	isBusyCopying sync.Mutex
```

Comment:
Strange name for a mutex -- copyMutex...?

Reply:
Not part of this PR... and I kind of like it since it doesn't have the type name in the variable name. :-)